node/ci/build.sh

```diff
@@ -3,6 +3,11 @@
 CI_DIR="$( cd "$( dirname "$0" )" && pwd )"
 
 pushd "$CI_DIR/.."
-sudo chmod -R 777 target
-cargo build --all --lib --bins --release --verbose --features "masq_lib/no_test_share automap/no_test_share"
+if [[ "$1" == "clear" ]]; then
+  sudo chmod -R 777 target
+fi
+cargo build --all --lib --bins --release --verbose --features masq_lib/no_test_share
```

Comment:
I think even automap/no_test_share had a value even though small. It eliminates some small piece of test code. otherwise bundled with the sharp (maybe just bad use of word coming from my first language where we say a 'sharp' version meaning the production one) binaries.

Reply:
I don't understand what you want me to do.

Reply:
My pardon. I talked about a justification that it would suit it better if you kept the other feature that you removed with your last change.

Before it was like  --features "masq_lib/no_test_share automap/no_test_share"